tests/unit/algorithms/action/adapters/mmaction/utils/test_action_config_utils.py

```diff
@@ -42,7 +43,7 @@ def test_patch_config() -> None:
     """
 
     cls_datapipeline_path = "otx/algorithms/action/configs/classification/x3d/data_pipeline.py"
-    work_dir = "OTX-tempdir9104"
+    work_dir = tempfile.mkdtemp("OTX-tempdir9104")
```

Comment:
```suggestion
    work_dir = tempfile.TemporaryDirectory("OTX-tempdir9104")
```

How about using `TemporaryDirectory` instead of `mkdtemp`?
It is deleted automatically on completion of the context or destruction of the temporary directory object.

Reply:
This passes a strange path in a different form than intended. I'll create a PR to fix it soon.